doc/en/trace.rst

```diff
@@ -0,0 +1,54 @@
+.. _Trace:
+
+Tracing Impacted Tests
+**********************
+
+Introduction
+============
+
+There are certain circumstances that one need to pay extra attentions to changes
```

Comment:
Let's focus on the behavior of this feature rather than the use case, because the use case is rather complicated to explain and we don't provide the the diff-to-json script here so it's not a full story. We can mention the possible usecase briefly in the end.